examples/library/uwsgi_log_plugin.py

```diff
@@ -30,7 +30,7 @@
 
 
 def import_from_uwsgi_log(filename):
-    fields = UWSGI_FIELDS.keys()
+    fields = list(UWSGI_FIELDS.keys())
```

Comment:
I'm not sure we need all these `list()`. Could you please check it? They also happen in other files such as `rows/cli.py`.